frontend/src/config/mozambique/prism.json

```diff
@@ -2,7 +2,8 @@
   "country": "Mozambique",
   "defaultLanguage": "en",
   "anticipatoryActionDroughtUrl": "https://data.earthobservation.vam.wfp.org/public-share/aa/aa_probabilities_triggers_moz.csv",
-  "anticipatoryActionStormUrl": " https://data.earthobservation.vam.wfp.org/public-share/aa/ts/outputs/latest.json.",
+  "anticipatoryActionDroughtStagingUrl": "https://data.earthobservation.vam.wfp.org/public-share/aa/staging/aa_probabilities_triggers_moz.csv",
+  "anticipatoryActionStormUrl": "https://data.earthobservation.vam.wfp.org/public-share/aa/ts/outputs/latest.json.",
```

Comment:
@wadhwamatic I'll merge like that for now but it should be relatively straightforward to make it work for storms (and flood) as well in the future.